frontend/src/translations/ru.global.json

```
@@ -0,0 +1,551 @@
+        },
+        "Transfer": {
+          "transferred": "Переведено ",
+          "to": "  "
+        },
+        "Stake": {
+          "staked": "В стейкинге: ",
+          "with_key": "с использованием публичного ключа ${public_key}..."
+        },
+        "AddKey": {
+          "new_key_added": "Добавлен новый ключ для ",
+          "with_permission_and_nounce": "с разрешением ${permission} и одноразовым номером ${nonce}",
+          "access_key_added_for_contract": "Ключ доступа добавлен для контракта ",
+          "with_permission_call_method_and_nounce": "с разрешением для вызова ${methods} методов с одноразовым номером ${nonce}",
+          "any": "любой"
+        },
+        "DeleteKey": {
+          "key_deleted": "Ключ удалён: ${public_key}..."
+        }
+      },
+      "TransactionAction": {
+        "batch_transaction": "Пакетная транзакция"
+      },
+      "ActionRowBlock": {
+        "by": "_"
+      },
+      "TransactionDetails": {
+        "signed_by": {
+          "title": "Подписавший аккаунт",
+          "text": "Аккаунт, который подписал и отправил транзакцию."
+        },
+        "receiver": {
+          "title": "Получатель",
+          "text": "Аккаунт, принимающий транзакцию."
+        },
+        "status": {
+          "title": "Статус",
+          "text": "Текущее состояние транзакции (в ожидании, успешно, сбой/завершено или не завершено). "
+        },
+        "transaction_fee": {
+          "title": "Комиссия за транзакцию",
+          "text": "Общая комиссия, уплаченная в NEAR за выполнение этой транзакции."
+        },
+        "deposit_value": {
+          "title": "Сумма депозита",
+          "text": "Сумма всех токенов NEAR, переведенных из подписывающего аккаунта в аккаунт получателя. Включает в себя токены, отправленные в передачах, и депозиты в вызовах функций."
+        },
+        "gas_used": {
+          "title": "Использовано газа",
+          "text": "Единицы газа, необходимые для выполнения этой транзакции."
+        },
+        "attached_gas": {
+          "title": "Прикреплённый газ",
+          "text": "Единицы газа, привязанные к транзакции (часто выше, чем «Использованный газ»)."
+        },
+        "created": {
+          "title": "Время оправки",
+          "text": "Отметка времени, когда эта транзакция была отправлена."
+        },
+        "hash": {
+          "title": "Хеш",
+          "text": "Уникальный идентификатор (хеш) этой транзакции."
+        },
+        "block_hash": {
+          "title": "Хеш блока",
+          "text": "Уникальный идентификатор (хеш) блока, в который была включена эта транзакция."
+        }
+      },
+      "ReceiptRow": {
+        "result": "Результат",
+        "no_result": "Без результата",
+        "empty_result": "Пустой результат",
+        "failure": "Сбой",
+        "success_receipt_id": "ID квитанции успешной транзакции",
+        "no_logs": "Нет записей",
+        "no_actions": "Нет действий"
+      }
+    },
```

```suggestion
    },
    "receipts": {
       "ReceiptAction": {
         "batch_receipt": "Пакетная Квитанция"
      },
    }
```

We missed this piece for other translations, so would you mind to add this to `en.global.json` file too? After then you need to update a component https://github.com/near/near-explorer/blob/master/frontend/src/components/receipts/Receipts.tsx#L34. @telezhnaya nice catch!